src/main/webapp/script.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Redirect to User Profile Page
+ */
+function goToUser() {
```

Comment:
This function and `goToUser()` are failing the checks because "they're defined but never used". However, they are actually being used in the project.html file.

Reply:
This might have absolutely nothing to do with it but I had an issue with assigning href to buttons vs. anchors and when I switched an element to an anchor it often worked. This looks completely correct but I'll do a deeper dive and see if I can find any potential issues.